client/app/scripts/superdesk/ui/ui.js

```diff
@@ -331,8 +331,23 @@ define([
                         css.right = Math.max(5, window.innerWidth - pos.left) + 'px';
 
                         dropdown.dropdownMenu.css({opacity: '0'}); // avoid flickering
-                        scope.$applyAsync(function() {
+
+                        scope.$applyAsync(function () {

```

Comment:
Well it does that already, the function is executed first... but yeah, never mind, better to focus on the tests :)